assistants/guided-conversation-assistant/pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "assistant"
 version = "0.1.0"
-description = "Demonstrates how to guide a user towards a goal using the guided-conversation library."
+description = "An assistant that will guide users through a conversation towards a specific goal."
```

Comment:
To be fair, this had been an intentional choice due to the reason why we were building it, why we're pointing folks to it, etc.  It is a demo of the guided conversation work.  That said, I guess nobody really sees this file so not sure it matters.